proj4/meta.yaml

```diff
@@ -7,10 +7,11 @@ source:
   url: http://download.osgeo.org/proj/proj-4.9.1.tar.gz
 
 build:
+  number: 0  # [osx]
   number: 1  # [not osx]
 
-requirements:
-  run:
+requirements:  # [win]
+  run:  # [win]
```

Comment:
I investigated this a little bit further and latest `obvious-ci`+`conda-build < 1.18.1` is OK with empty fields.
Conda `conda-build==1.18.1` returns an error when `obvious-ci` tries this:

```python
Traceback (most recent call last):
  File "/Users/travis/miniconda/bin/obvci_conda_build_dir.py", line 19, in <module>
    Builder.handle_args(args).main()
  File "/Users/travis/miniconda/lib/python2.7/site-packages/obvci/conda_tools/build_directory.py", line 232, in main
    recipe_metas = self.fetch_all_metas()
  File "/Users/travis/miniconda/lib/python2.7/site-packages/obvci/conda_tools/build_directory.py", line 206, in fetch_all_metas
    recipe_metas = sort_dependency_order(recipe_metas)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/obvci/conda_tools/build_directory.py", line 85, in sort_dependency_order
    meta.get_value('requirements/build', []))
TypeError: coercing to Unicode: need string or buffer, list found
```